### PR TITLE
Social: Display the share counter. 

### PR DESCRIPTION
**Pull request by spsiddarthan**
We are going to be displaying the number of shares made in the past 30 days. This PR is a WIP and currently displays the share counter by calling the API, however it does that on the old design we had. I will work on updating the design here, next. 

#### Testing instructions:

* Go to the Jetpack Social admin page and observe that the share count shown to you is accurate. 
* Test for responsiveness by viewing the page in different screen resolutions. 

https://user-images.githubusercontent.com/6594561/175268756-91e658aa-87f1-461b-999b-bb71b0530cc0.mov

### CODE REVIEW

File: projects/plugins/social/src/js/components/header/index.js [hunk 1]
```diff
@@ -1,23 +1,61 @@
...
+				<a href={ link } className={ styles.action }>
+					{ icon && <Icon icon={ icon } size={ 16 } /> }
+					{ label }
+				</a>
+			</Text>
+		) ) }
+	</div>
 );
+const Header = () => {
+	useDispatch( STORE_ID ).getSharesCount();
```
Comment (pablinos) [inline projects/plugins/social/src/js/components/header/index.js]: If this is to live here then I think it needs to be wrapped in a `useEffect`
```suggestion
	useEffect( () => { useDispatch( STORE_ID ).getSharesCount() }, [] );
```

Without it the component will be rendered again as the shares count is received and it will get stuck in a loop requesting the shares count constantly.

I think it's good to have this in the store and a way to request it, but I wonder if for now we should add this into the initial state? Maybe something for a follow-up PR

File: projects/plugins/social/src/js/components/header/index.js [hunk 2]
```diff
@@ -1,23 +1,61 @@
...
+		<Container horizontalSpacing={ 3 } horizontalGap={ 7 } className={ styles.container }>
+			<Col sm={ 4 } md={ 4 } lg={ 5 }>
+				<H3 mt={ 2 }>{ __( 'Post everywhere at any time', 'jetpack-social' ) }</H3>
+				<Actions actions={ actions } />
+			</Col>
+			<div className={ styles.sharesCard }>
+				<div className={ styles.socialIcon }>
+					<SocialIcon />
+				</div>
+				<div className={ styles.sharesCardHeading }>Total shares this month</div>
```
Comment (jeherve) [inline projects/plugins/social/src/js/components/header/index.js]: You'll need to translate this as well.

```suggestion
				<div className={ styles.sharesCardHeading }>{ __( 'Total shares this month', 'jetpack-social' ) }</div>
```

File: projects/plugins/social/composer.json [hunk 3]
```diff
@@ -72,6 +72,6 @@
 			"automattic/jetpack-autoloader": true,
 			"automattic/jetpack-composer-plugin": true
 		},
-		"autoloader-suffix": "c4802e05bbcf59fd3b6350e8d3e5482c_socialⓥ1_0_1_alpha"
+		"autoloader-suffix": "c4802e05bbcf59fd3b6350e8d3e5482c_socialⓥ1_1_0_alpha"
```
Comment (jeherve) [inline projects/plugins/social/composer.json:75]: To avoid such unrelated changes, I would recommend that the next time you push a new version of the plugin, you immediately create a new PR that just bumps the version number of the plugin to start a new release cycle.

Comment (spsiddarthan) [reply projects/plugins/social/composer.json:75]: Thanks for the pointer, @jeherve! We will make sure we keep this in mind.

Comment (pablinos) [reply projects/plugins/social/composer.json:75]: Weirdly, I thought we already did that @sdixon194 mentioned doing that as we were going through the process.

Maybe we just bumped it to 1.0.1, and that's not significant enough?

Comment (jeherve) [reply projects/plugins/social/composer.json:75]: > Maybe we just bumped it to 1.0.1, and that's not significant enough?

Ah yes you're right, that was it!

Comment (pablinos) [reply projects/plugins/social/composer.json:75]: Cool, does that mean in future we should bump the minor version number? I'll add that to the notes.

**Review by pablinos — APPROVED** [review]
This looks good. We might want to move to using initial state, and update the CSS, but we can do that in follow up PRs
